Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+import SwiftUI
+
+public typealias SliderValue = BinaryFloatingPoint & Strideable & CustomStringConvertible
+
+/// Enables selection from a range of values.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/slider/)
+public struct Slider<Value>: View where Value: SliderValue, Value.Stride: BinaryFloatingPoint {
+
+    enum SelectedValue {
+        case single(Binding<Value>?)
+        case range(Binding<ClosedRange<Value>>?)
+    }
+
+    @Environment(\.sizeCategory) var sizeCategory
+
+    let value: SelectedValue
+    let valueLabel: String
+    let startLabel: String
+    let endLabel: String
+    let bounds: ClosedRange<Value>
+
+    @State private var isFirstThumbDrag = false
+    @State private var isSecondThumbDrag = false
+    @State private var firstValue: Value {
+        didSet {
+            switch value {
+                case .single(let currentValue):
+                    currentValue?.wrappedValue = firstValue
+                case .range:
+                    updateSelectedRangeIfNeeded()
+            }
+        }
+    }
+
+    @State private var secondValue: Value? {
+        didSet {
+            updateSelectedRangeIfNeeded()
+        }
+    }
+
+    @State private var dragStartOffset = 0.0
+
+    private let buckets: [Range<Value>]
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(valueLabel)
+                .foregroundColor(.blueNormal)
+                .padding(.bottom, .medium)
+                .accessibility(.sliderValueLabel)
+
+            SingleAxisGeometryReader(axis: .horizontal, alignment: .bottom) { width in
+                track
+                    .overlay(
+                        controls(sliderWidth: width)
+                    )
+                    .padding(.vertical, thumbVerticalOverlapping)
+            }
+
+            HStack {
```

Comment:
The stack is missing both spacing and alignment. The Spacer also misses the `minLength` property. This would result in a layout that is out of our control.